scope/src/models/core.rs

```diff
@@ -19,6 +19,8 @@ pub struct ModelMetadataAnnotations {
     /// Directory containing the resource, generated automatically.
     pub file_dir: Option<String>,
     #[serde(rename = "scope.github.com/bin-path")]
+    /// When running commands, the working directory of the command
```

Comment:
You'll want/need to move the `bin-path` line to be above the bin_path variable, so it can be deserialzec correctly.